<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="dDayTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>741</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>254</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="background">
      <color alpha="0">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D-Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Progress</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>481</width>
      <height>741</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(64, 255, 112);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>D-Day</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>450</y>
        <width>251</width>
        <height>241</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_AddDay">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/btn/D-Day icon/buttons/add.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_AddDay">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;Add D-Day&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_DelDay">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/btn/D-Day icon/buttons/negative.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_DelDay">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff2e66;&quot;&gt;Delete D-Day&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_EditDay">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/btn/D-Day icon/buttons/edit button2.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_AddDay_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffaa00;&quot;&gt;Edit D-Day&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QProgressBar" name="pgBar_FirstPage">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_DDayName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;D-Day Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_DdayDisplay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>411</width>
        <height>241</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>411</width>
        <height>281</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;D+0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>451</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/bgi/D-Day icon/background_imege/post-it4.png);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_PgPlan">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Progress of Plan&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>graphicsView</zorder>
     <zorder>frame</zorder>
     <zorder>pgBar_FirstPage</zorder>
     <zorder>lbl_DDayName</zorder>
     <zorder>lbl_DdayDisplay</zorder>
     <zorder>lbl_PgPlan</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Detail Plan List</string>
     </attribute>
     <widget class="QPushButton" name="btn_AddPlan">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/btn/D-Day icon/buttons/plus2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_DelPlan">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>340</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/btn/D-Day icon/buttons/remove.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_AddPlan">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#000000;&quot;&gt;Add Plan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_DelPlan">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff2e66;&quot;&gt;Delete Plan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="pgBar_SecondPage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="memoEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>451</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(255, 255, 255, 0));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Please enter this D-Day's memo</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>381</y>
        <width>451</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/bgi/D-Day icon/background_imege/post-it2.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_MemoSave">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>660</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/btn/D-Day icon/buttons/save.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_SaveMemo">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>660</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#000000;&quot;&gt;Save Memo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ClrMemo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>660</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff2e66;&quot;&gt;Clear Memo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_MemoClr">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>660</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/btn/D-Day icon/buttons/eraser.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="planListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <zorder>btn_AddPlan</zorder>
     <zorder>btn_DelPlan</zorder>
     <zorder>lbl_AddPlan</zorder>
     <zorder>lbl_DelPlan</zorder>
     <zorder>pgBar_SecondPage</zorder>
     <zorder>graphicsView_2</zorder>
     <zorder>memoEdit</zorder>
     <zorder>btn_MemoSave</zorder>
     <zorder>lbl_SaveMemo</zorder>
     <zorder>lbl_ClrMemo</zorder>
     <zorder>btn_MemoClr</zorder>
     <zorder>planListWidget</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveData"/>
    <addaction name="actionLoadData"/>
   </widget>
   <widget class="QMenu" name="menuForm">
    <property name="title">
     <string>D-Day</string>
    </property>
    <addaction name="actionAddDday"/>
    <addaction name="actionDelDday"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menuForm"/>
  </widget>
  <action name="actionSaveData">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoadData">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionAddDday">
   <property name="text">
    <string>Add D-Day</string>
   </property>
  </action>
  <action name="actionDelDday">
   <property name="text">
    <string>Delete D-Day</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add Plan</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete Plan</string>
   </property>
  </action>
  <action name="actionAll_Clear">
   <property name="text">
    <string>All Clear</string>
   </property>
  </action>
  <action name="actionFont_2">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionFont_3">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionFont_4">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
